testing/jest/button.tests.tsx

```diff
@@ -210,15 +210,46 @@ describe('Button', () => {
         });
 
         describe('contentRender', () => {
-            it('should render template', () => {
+            const contentRender = ({ text }) => <div className={'custom-content'}>{text + 123}</div>;
+
+            it('should render contentRender', () => {
                 const button = render({
                     text: 'My button',
-                    contentRender: ({ text }) => <div className={'custom-content'}>{text + 123}</div>,
+                    contentRender,
                 });
-                const buttonContentChildren = button.find('.dx-button-content').children();
+                const customRender = button.find(contentRender);
+
+                expect(customRender.exists()).toBe(true);
+                expect(customRender.exists('.custom-content')).toBe(true);
+
+                expect(customRender.props().text).toBe('My button');
+                expect(customRender.render().text()).toBe('My button123');
+            });
+
+            it('should rerender contentRender in runtime', () => {
+                const button = mount(<Button text='My button' />);
+                
+                expect(button.exists(contentRender)).toBe(false);
+
+                button.setProps({ contentRender });
+                expect(button.exists(contentRender)).toBe(true);
+                
+                button.setProps({ contentRender: undefined });
+                expect(button.exists(contentRender)).toBe(false);
+            });
+
+            it('should change properties in runtime', () => {
+                const button = mount(<Button text='My button' contentRender={contentRender} />);
+                let buttonContent = button.find(contentRender);
+                
+                expect(buttonContent.props().text).toBe('My button');
+                expect(buttonContent.render().text()).toBe('My button123');
+
+                button.setProps({ text: 'New value' });
+                buttonContent = button.find(contentRender);
 
-                expect(buttonContentChildren.props().text).toBe('My button');
-                expect(buttonContentChildren.render().text()).toBe('My button123');
+                expect(buttonContent.props().text).toBe('New value');
+                expect(buttonContent.render().text()).toBe('New value123');
```

Comment:
Why we should call `render` method manually?

Reply:
Remained from previous tests.
Refactored.